docs/asciidoc/advancedFeatures.adoc

```diff
@@ -624,6 +624,18 @@ invoked whenever a `Runnable` task submitted to a `Scheduler` throws an `Excepti
 that if there is an `UncaughtExceptionHandler` set for the `Thread` that ran the task,
 both the handler and the hook are invoked).
 
+[[scheduler-virtual-thread]]
```

Comment:
Please review section 4.5 and consider updating the boundedElastic description instead of adding a new one here. The remark about newBoundedElastic can become part of the above section for factories, so it offers a guidance to the user how to replace the old behaviour with the new one in places that they need a non-global instance.